carts/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from carts.views import CartView
+
+urlpatterns = [
+    path('/list', CartView.as_view()),
+    path('/list/<int:user_id>', CartView.as_view()),
+    path('/list/<int:cart_id>/delete', CartView.as_view()),
+]
```

Comment:
```suggestion
    path('', CartView.as_view()),
    path('/<int:user_id>', CartView.as_view()),
    path('/<int:cart_id>', CartView.as_view()),
]
```
1. carts/list 가 최종 url이 되는데, carts로 충분히 cart들을 표현 합니다.
2. RESTful 규칙에 의거하면 url은 자원만 나타냅니다. 행위 즉 동사는 http method로 표시합니다. 따라서 delete는 지워주세요.

Reply:
의미가 중복되는 url 삭제 및 HTTP Method나 동사표현이 url에 들어가면 안 된다는 것을 배웠습니다.
경훈님 RESTful API 규칙을 보니 제가 한게 딱 잘못된 사례로 나와있네요 감사합니다!